examples/ssoadmin/permissionsetinlinepolicy.yaml

```diff
@@ -6,21 +6,22 @@ metadata:
     upjet.upbound.io/manual-intervention: "This resource requires a valid instanceArn(The Amazon Resource Name (ARN) of the SSO Instance under which
     the operation will be executed)."
   labels:
-    testing.upbound.io/example-name: example
-  name: example
+    testing.upbound.io/example-name: ssoadmin-permission-set-inline-policy
+  name: ssoadmin-permission-set-inline-policy
 spec:
   forProvider:
-    instanceArn: ${tolist(data.aws_ssoadmin_instances.example.arns)[0]}
+    instanceArnSelector:
+      matchLabels:
+        testing.upbound.io/example-name: ssoadmin-permission-set-inline-policy
```

Comment:
We may need to remove these lines after reference verification?

Reply:
I'm not sure what you mean, but maybe my answer at https://github.com/upbound/provider-aws/pull/928#discussion_r1370683283 clarifies?